src/logger/index.ts

```diff
@@ -12,65 +12,74 @@ export const LogLevels: { [level: string]: LogLevel } = {
   NONE: 'NONE'
 };
 
+const logLevelRanks = {
+  DEBUG: 1,
+  INFO: 2,
+  WARN: 3,
+  ERROR: 4,
+  NONE: 5
+};
+
```

Comment:
Given the fact that we are using TS, you could implement this Object as a [const enum](https://www.typescriptlang.org/docs/handbook/enums.html#const-enums)

Reply:
In fact ... I'm not sure about this but you are basically rewriting [this](https://github.com/splitio/javascript-commons/blob/logger_polishing_log_messages/src/types.ts#L53). and you are doing something similar [here](https://github.com/splitio/javascript-commons/blob/logger_polishing_log_messages/src/logger/index.ts#L7-L13).
Maybe what you could do is remove the String Literal typing from the `types.ts` and replaced with a Const Enum definition there. Let's discuss this in a call

Reply:
It is a great idea using const enums, but after trying it, there is a few places where a need to access enum values dynamically and it is not allowed.